app/recordtransfer/forms/submission_forms.py

```diff
@@ -698,7 +698,7 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super(RightsFormSet, self).__init__(*args, **kwargs)
-        self.forms[0].empty_permitted = False
+        self.forms[0].empty_permitted = True  # Changed to True to make it optional
```

Comment:
This comment is unnecessary

```suggestion
        self.forms[0].empty_permitted = True
```